versions/3.0.md

```diff
@@ -2069,7 +2069,7 @@ value), references MAY also be made through a relative `operationRef`:
 links:
   UserRepositories:
     # returns array of '#/components/schemas/repository'
-    operationRef: '#/paths~12.0~1repositories~1/{username}/get'
+    operationRef: '#/paths/~12.0~1repositories~1/{username}/get'
```

Comment:
Looking at it again, it seems this might need to be `#/paths/~12.0~1repositories~1{username}/get`?
That would translate to:

```yaml
paths:
  /2.0/repositories/{username}:
    get:  
```